README.md

```diff
@@ -1,3 +1,17 @@
 # The DMV
 
-This is the starter repo for the BE Mod1 DMV project.
+# BE MOD 1 DMV PROJECT - ANDY STILMOCK
+
+## Digging into the existing codebase
+
+  ### - Describe the steps you took to dig in to this code base. What was your process? If you don’t feel you had a process, define one that you might like to try next time.
+Initially, I began with reading through the iteration 1 directions before diving into the code in order to get a good grip on the expectations and setup for this project. After cloning the repo to my computer and diving into the preexisting files - I just started at the top of the list in the library files, looking at the library file then the corresponding test file in the spec folder. As I went through the files I took some notes on what I was confident I would be able to figure out/complete for the steps listed in the interaction pattern but also, took notes on the lines of code that I did not understand or knew I would need some further questioning and research on to take a look at when I started working through the tests and interaction patterns for the project. 
```

Comment:
Love the process! Especially the part about reading the directions to understand the expectations first.